data/replaced_tags.txt

```diff
@@ -149,8 +149,6 @@ artwork_type=bust : artwork_type=statue
 
 place=allotments : place=neighbourhood
 
-crossing=traffic_signals : highway=traffic_signals
```

Comment:
Crossing traffic lights will be ignored now, correct? They should be added separately, right? @euf can we do a red + green icon for crossings?

Reply:
Like a pedestrian oriented traffic lights you mean? Sure we can 😄